src/content/configuration/node.md

```diff
@@ -50,14 +50,18 @@ Since webpack 3.0.0, the `node` option may be set to `false` to completely turn
 
 ## `node.global`
 
-`boolean = false`
+`boolean = true`
+
+Is `false` for [targets](https://webpack.js.org/configuration/target/) `node`, `async-node` and `electron-main`.
```

Comment:
Is it defaulting to `false` for these targets, or is it just `false` and you cannot override? If its defaulting to but still can be overridden by setting it to a different value, we should say so explicitly for all three occasions like so:
```suggestion
Defaults to `false` for [targets](https://webpack.js.org/configuration/target/) `node`, `async-node` and `electron-main`.
```

Reply:
It is defaulting to `false` and will be overridden when specified in the webpack-config. Your suggested wording is more explicit, I added it to the commit. :+1:

Reply:
thank you! i didnt have time to double check so just asked :D